.github/workflows/ci.yml

```
@@ -91,14 +91,15 @@ jobs:
       - run: pip install coverage[toml]
 
       - name: download coverage data
-        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        uses: actions/download-artifact@7a1cd3216ca9260cd8022db641d960b1db4d1be4 # v4.0.0
         with:
-          name: coverage-data
+          path: all-artifacts/
```

Nitpick, descriptive:

```suggestion
          path: all-coverage/
```

---

I'm going to leave this unless you insist as the contents really are "all artifacts": if someone in the future adds another artifact upload earlier in the workflow, the unrelated artifact will show up in this directory. So any code that processes "all-artifacts" should be aware of that.

---

Sure, works for me.